test/fixtures/dialects/hive/double_equals.sql

```diff
@@ -0,0 +1,6 @@
+SELECT
+    a
+FROM
+    t
+WHERE
+    t.a = = t.b
```

Comment:
Can you add a test case without the space? I just want to make sure this doesn't actually require a lexer change

Reply:
I replaced the spaced version as that one seemed odd (and was inadvertent). Let me know if you want both test cases.